chess/frontend/gui/src/modules/Common/utils.js

```diff
@@ -9,7 +9,8 @@ export const checkSignedIn = () => {
       'Authorization': `JWT ${localStorage.getItem('token')}`
     },
     error: () => {
-      if (!(['/', '/sign_in', '/sign_in/', '/create_user', '/create_user/'].some(path => path === window.location.pathname))) {        
+      if (!(['/', '/sign_in', '/create_user'].some(path => 
+          (path !== '/' && path.includes(window.location.pathname)) || (path === '/' && path === window.location.pathname)))) {      
```

Comment:
I truly have no idea what this is trying to do. Can you explain it for me?

Reply:
So if the path is '/' I am checking if it is an exact match '===' but if it is anything else I want to just see if it includes it, so /games/13 , or /sign_in , will all be fine since I am using includes and not exact match

Reply:
And that's the most terse way to express that in JS? May God have mercy on us all